src/app/containers/StoryPromo/index.jsx

```diff
@@ -105,7 +106,7 @@ const StoryPromoContainer = ({
   const { pageType } = useContext(RequestContext);
   const handleClickTracking = useCombinedClickTrackerHandler(eventTrackingData);
 
-  const linkId = getUniqueLinkId(item, labelId);
+  const linkId = buildUniquePromoId(labelId, item, index); // TODO add promo index here
```

Comment:
```suggestion
  const linkId = buildUniquePromoId(labelId, item, index);
```

Just got a little comment still in code 👍

Reply:
Nice spot!